phy/stats/ccg.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+"""Cross-correlograms."""
+
+#------------------------------------------------------------------------------
+# Imports
+#------------------------------------------------------------------------------
+
+import numpy as np
+
+from ..ext import six
+from ..utils.array import _index_of, _unique
+
+
+#------------------------------------------------------------------------------
+# Cross-correlograms
+#------------------------------------------------------------------------------
+
+def _increment(arr, indices):
+    """Increment some indices in a 1D vector of non-negative integers.
```

Comment:
We could do that but I fear that it would badly affect performance. This is called during a performance-critical loop during the computations of CCGs...